containers/images/core/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM centos:7
+
+ENV RAILS_ENV="production"
```

Comment:
What's the benefit of a source based container if not able to run in development mode?

Reply:
I can get a PR or a different branch to run in production, imho great for testing a PR or running it on a patched production env